src/main/platform.h

```diff
@@ -26,7 +26,7 @@
 #pragma GCC poison sprintf snprintf
 #endif
 
-#if defined(STM32F745xx) || defined(STM32F746xx) || defined(STM32F722xx)
+#if defined(STM32F765xx) || defined(STM32F745xx) || defined(STM32F746xx) || defined(STM32F722xx)
```

Comment:
Minor niggle, but should we order these?

Reply:
Sure, why not